kanidm_book/src/server_configuration.md

```diff
@@ -2,76 +2,97 @@
 
 ### Configuring server.toml
 
-You will also need a config file in the volume named `server.toml` (Within the container it should be `/data/server.toml`). Its contents should be as follows:
+You need a configuration file in the volume named `server.toml`. (Within the container it should be `/data/server.toml`.) Its contents should be as follows:
 
-    #   The webserver bind address. Will use HTTPS if tls_* is provided.
+    #   The webserver bind address. Will use HTTPS if tls_* 
+    #   is provided.
     #   Defaults to "127.0.0.1:8443"
     bindaddress = "[::]:8443"
     #
-    #   The read-only ldap server bind address. The server will use LDAPS if tls_* is provided.
+    #   The read-only ldap server bind address. The server 
+    #   will use LDAPS if tls_* is provided.
     #   Defaults to "" (disabled)
     # ldapbindaddress = "[::]:3636"
     #
     #   The path to the kanidm database.
     db_path = "/data/kanidm.db"
     #
-    #   If you have a known filesystem, kanidm can tune sqlite to match. Valid choices are:
+    #   If you have a known filesystem, kanidm can tune sqlite
+    #   to match. Valid choices are:
     #   [zfs, other]
-    #   If you are unsure about this leave it as the default (other). After changing this
+    #   If you are unsure about this leave it as the default 
+    #   (other). After changing this
     #   value you must run a vacuum task.
     #   - zfs:
-    #     * sets sqlite pagesize to 64k. You must set recordsize=64k on the zfs filesystem.
+    #     * sets sqlite pagesize to 64k. You must set 
+    #   recordsize=64k on the zfs filesystem.
```

Comment:
For this and similar points below - if we're going to line break something two nested indents deep, can we please indent them to align with its parent line? It's a little hard to follow otherwise.